Casks/adoptopenjdk10.rb

```diff
@@ -3,7 +3,7 @@
   sha256 '550fac840563073dcfaa1ffd4a3a9cc68fd8ccc71bbc65baf3cb32ec734c30dd'
 
   # github.com/AdoptOpenJDK was verified as official when first introduced to the cask
-  url "https://github.com/AdoptOpenJDK/openjdk#{version.before_comma}-binaries/releases/download/jdk-#{version.before_comma}.#{version.after_comma.before_colon}%2B#{version.after_colon}/OpenJDK#{version.before_comma}U-jdk_x64_mac_hotspot_#{version.before_comma}.#{version.after_comma.before_colon}_#{version.after_colon}.tar.gz"
+  url 'https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk-10.0.2%2B13.1/OpenJDK10U-jdk_x64_mac_hotspot_10.0.2_13.tar.gz'
```

Comment:
@gdams 
It's sufficient to change version '10,0.2:13' to version '10,0.2:13.1'

Reply:
right URL for binary is
`https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk-10.0.2%2B13.1/OpenJDK10U-jdk_x64_mac_hotspot_10.0.2_13.tar.gz`

```diff
diff --git a/Casks/adoptopenjdk10.rb b/Casks/adoptopenjdk10.rb
index 08ff497..68979f8 100644
--- a/Casks/adoptopenjdk10.rb
+++ b/Casks/adoptopenjdk10.rb
@@ -1,9 +1,9 @@
 cask 'adoptopenjdk10' do
-  version '10,0.2:13'
+  version '10,0.2:13.1'
   sha256 '550fac840563073dcfaa1ffd4a3a9cc68fd8ccc71bbc65baf3cb32ec734c30dd'
 
   # github.com/AdoptOpenJDK was verified as official when first introduced to the cask
-  url "https://github.com/AdoptOpenJDK/openjdk#{version.before_comma}-binaries/releases/download/jdk-#{version.before_comma}.#{version.after_comma.before_colon}%2B#{version.after_colon}/OpenJDK#{version.before_comma}U-jdk_x64_mac_hotspot_#{version.before_comma}.#{version.after_comma.before_colon}_#{version.after_colon}.tar.gz"
+  url "https://github.com/AdoptOpenJDK/openjdk#{version.before_comma}-binaries/releases/download/jdk-#{version.before_comma}.#{version.after_comma.before_colon}%2B#{version.after_colon}/OpenJDK#{version.before_comma}U-jdk_x64_mac_hotspot_#{version.before_comma}.#{version.after_comma.before_colon}_13.tar.gz"
   appcast "https://github.com/adoptopenjdk/openjdk#{version.before_comma}-binaries/releases.atom"
   name 'AdoptOpenJDK 10'
   homepage 'https://adoptopenjdk.net/'
```

something like this might be good as a temporary fix

Reply:
verified
```bash
$ brew cask audit --download Casks/adoptopenjdk10.rb
==> Downloading https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk-10.0.2%2B13.1/OpenJDK10U-jdk_x64_mac_hotspot_10.0.2_13.tar.gz
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws.com/140418978/8b643480-f247-11e8-8bee-38e0f2e5dfd1?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2
########                                                                  11.9%
```